components/guides/GuideList.tsx

```diff
@@ -6,7 +6,7 @@ function GuideList({ listItems }) {
   return (
     <ul>
       {listItems.map((item) => (
-        <li key={item.text}>{item.text}</li>
+        <li key={item.text} dangerouslySetInnerHTML={{ __html: item.text }} />
```

Comment:
Please never use `dangerouslySetInnerHTML`. It is dangerous because we don't really trust translators.

Use `<Trans>` component instead. (see examples by searching it in the codebase)

Reply:
Good point